ceph/rados/core_workflows.py

```diff
@@ -2740,3 +2740,52 @@ def get_host_object(self, hostname):
 
         log.error(f"Could not find host object for host {hostname}")
         return None
+
+    def set_autoscale_flag(self, retry: int = 15) -> bool:
```

Comment:
There exists a module to take care of pg_autoscaler `configure_pg_autoscaler(self, **kwargs)`, any possibility that both can be merged?

Reply:
the functionality of both the modules is different. 
`configure_pg_autoscaler` deals with the pool level settings to change the autoscale state on the pool.

This method is a global level flag, like noout, noscrub etc, that acts on entire cluster. I do not feel that the two should be merged into one function.